### PR TITLE
Document Airzone Cloud zone switch entities

### PR DESCRIPTION
**Pull request by Noltari**
## Proposed change
Document Airzone Cloud zone switch entities.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/125917
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced support for a "Switch" entity in the Airzone integration, enhancing control options for HVAC systems.
	- Updated documentation to include a new section on creating switch entities for Airzone zones.

### CODE REVIEW

File: source/_integrations/airzone_cloud.markdown [hunk 1]
```diff
@@ -108,6 +110,10 @@ For each Airzone WebServer (HVAC Wi-Fi controller), the following *sensors* are
 | :------------------ | :------------------------------------------------- |
 | rssi                | Wi-Fi RSSI.                                        |
 
+## Switch
+
+For each Airzone zone (thermostat), a switch entity is created.
```
Comment (c0ffeeca7) [inline source/_integrations/airzone_cloud.markdown]: What does this switch do? Turn the thermostat on and off?

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/airzone_cloud.markdown (hunk 1 above)
Comment (Noltari) [reply source/_integrations/airzone_cloud.markdown]: > What does this switch do? Turn the thermostat on and off?

Yes, it turns it on/off without changing the HVAC mode.

Comment (c0ffeeca7) [inline source/_integrations/airzone_cloud.markdown]:
```suggestion
For each Airzone zone (thermostat), a switch entity is created to turn the thermostat on or off (without changing the HVAC mode).
```
something like that?

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @Noltari :+1:

**Review by frenck — APPROVED** [review]

**Pull request merged**